tutorials/Flop_Count_Example.ipynb

```diff
@@ -0,0 +1,364 @@
+  {
+   "cell_type": "markdown",
+   "id": "170ea68b",
+   "metadata": {},
+   "source": [
+    "Most common ML operations (e.g. matrix multiplications, convolutions, elementwise arithmetic operations, etc.) already have registered counter functions. Other ops, like sigmoid and tanh, whose flop count can vary by platform, are by default ignored, but have estimated counter functions that can be optionally turned on (see the Single-Flop Estimate section for details). \n",
+    "\n",
+    "If you need to add support for more ops or override the counter's default implementation, read on!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "322a5695",
+   "metadata": {},
+   "source": [
+    "## Advanced Usage"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7b5e4ff1",
+   "metadata": {},
+   "source": [
+    "### How the Counter Works\n",
+    "\n",
+    "The flop counter in `neuralcompression` makes heavy use of the counting utilities in [fvcore](https://github.com/facebookresearch/fvcore). Counting a model's flops is a two-step process:\n",
+    "\n",
+    "1. Using PyTorch's [TorchScript](https://pytorch.org/docs/stable/jit.html) capabilities, the model is first JIT-traced into a computational graph. Each node in the graph corresponds to an ATen (linear algebra) operation, like matrix multiplications, convolutions, and elementwise operations like additions/subtractions.\n",
+    "\n",
+    "\n",
+    "2. Every node in the traced graph is iterated over, and if a node is associated with a registered flop-counting function, that function is invoked and the counted flops are added to the model's total.\n",
+    "\n",
+    "\n",
+    "In this second step, the counter's registered flop functions are stored as a dictionary mapping operator names (e.g. `aten::add`, `aten::matmul`) to counter functions. These functions have a signature of:\n",
+    "\n",
+    "```\n",
+    "def my_counter_function(inputs: List[torch._C.Value], outputs: List[torch._C.Value]) -> float\n",
+    "```\n",
+    "\n",
+    "where objects of type `torch._C.Value` represent the symbolic inputs and outputs for the node in the graph (i.e. they're not actual concrete `Tensor`s). Check out the `fvcore` codebase for [examples of how to write counter functions](https://github.com/facebookresearch/fvcore/blob/master/fvcore/nn/jit_handles.py)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "034e4d71",
+   "metadata": {},
+   "source": [
+    "### What is Counted as a Flop\n",
+    "\n",
+    "Note that all of the built-in counter functions consider one fused multiply-accumulate (MAC) to be one flop. This means that a matrix multiplication between an `N x K` and a `K x M` matrix is recorded as `N * K * M` flops (since `N * K * M` MACs are performed). For elementwise arithmetic operations such as addition, subtraction, absolute value, etc., one flop is counted for each element of the output tensor. For example, adding two `N x K` matrices counts as `N * K` flops."
```

Comment:
Suggestion: Change second sentence to "For non-fused elementwise arithmetic operations..."